addons/sale/models/account_move.py

```diff
@@ -157,10 +157,12 @@ def _compute_partner_credit(self):
                 move.company_id,
                 move.date
             )
-            amount_to_invoice_currency = sale_order.currency_id._convert(
-                sale_order.amount_to_invoice,
-                move.company_currency_id,
-                move.company_id,
-                move.date
+            amount_to_invoice_currency = sum(
+                sale.currency_id._convert(
+                    sale.amount_to_invoice,
+                    move.company_currency_id,
+                    move.company_id,
+                    move.date
+                ) for sale in sale_order
```

Comment:
```suggestion
                sale_order.currency_id._convert(
                    sale_order.amount_to_invoice,
                    move.company_currency_id,
                    move.company_id,
                    move.date
                ) for sale_order in sale_orders
```
Tiny detail, but easier to read/understand IMO :slightly_smiling_face: